docs/manage/get_support.rst

```diff
@@ -36,22 +36,33 @@ In case you receive the ``database disk image is malformed`` error in Terminal,
 
 The ``sqlite3`` command is necessary. This can be installed with ``sudo apt install sqlite3`` on a Debian-based system.
 
-.. code-block:: bash
-
-        # Go to where your Kolibri data is stored
-        cd ~/.kolibri
-        # Create a directory to save the old DB in
-	mkdir -p malformed
-        # Copy the old malformed DB
-	cp -b db.sqlite3* malformed/
-        # Create a new database
-	sqlite3 db.sqlite3 .dump | sqlite3 fixed.db
-        # Move to the active location
-	mv fixed.db db.sqlite3
-        # Remove so-called "write-ahead log"
-	rm -f db.sqlite3-wal db.sqlite3-shm
-
-After this, you should be able to start Kolibri again.
+#. Open the Kolibri home folder in the Terminal.
+
+    .. code-block:: bash
+
+      cd ~/.kolibri
+
+#. Create a new directory and save the old database.
+
+    .. code-block:: bash
+
+      mkdir -p malformed
+      cp -b db.sqlite3* malformed/
+
+#. Create a new database and move it to the active location.
+   
+    .. code-block:: bash
+
+      sqlite3 db.sqlite3 .dump | sqlite3 fixed.db
+      mv fixed.db db.sqlite3
+
+#. Remove the following files.
+   
+    .. code-block:: bash
+
+      rm -f db.sqlite3-wal db.sqlite3-shm job_storage.sqlite3 /process_cache/cache.db
+
+#. Restart Kolibri.
```

Comment:
Just want to make sure that this is intentional: The steps above are no longer in a copy-paste friendly block. That's okay, copy-pasting long commands is a bit of a mal-practice, although I think these lines are all very short and readable.

But by not being in the same block, it's now implicit that everything follows `cd ~/.kolibri`, and cannot be run individually.

As Kolibri is likely crashed during this operation, we didn't put "start Kolibri" in the beginning, and finished it with " After this, you should be able to start Kolibri again." - restart makes slightly less sense when it's not running.

Reply:
The reason why I decided to split the block, is because while working on supporting the translation to French, I realized that none of the in-block-comments were being extracted as translatable strings, and would remain in English. I think we should aim to have the code comments translated too, and write them in the way that permits string extraction.

Since the commands are in an ordered list, my assumption was that it was clear enough the commands needed to be run in a sequence, and not individually. 

Do you recommend putting the explicit `stop` (at the beginning) and `start` commands at the end of the list?

Reply:
Ooooh that's why! I totally didn't realize that these code blocks could behave like this. Can whole blocks  be marked for translation?

Reply:
Well, commands should remain marked as untranslatable (in between pairs of back ticks or behind ``.. code-block::``), so the only way I see is to separate comments and code lines completely. This will prevent c/p friendly blocks, but as you say, that is not kosher anyways  :slightly_smiling_face:

Reply:
Thanks, I get what is at stake, let's not push code fragments together with translations, that seems wrong. The "hack" here was nice for non-translatable purposes otherwise, but not wort pursuing.